React/Views/RCTAlertController.m

```diff
@@ -0,0 +1,25 @@
+#import "RCTAlertController.h"
+
+@interface RCTAlertController ()
+
+@property (nonatomic, strong) UIWindow *alertWindow;
+
+@end
+
+@implementation RCTAlertController
+
+- (UIWindow *)alertWindow {
+    if (_alertWindow == nil) {
+        _alertWindow = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
```

Comment:
@devon94  One question: Should this be the key window size? 

The scenario I'm thinking about is iPad split screen. I think this would return the entire device screen size instead of the app's window.

Reply:
Ah yea, you're right! I didn't think about iPad/split screen when looking at this issue. The keyWindow bounds should fix that scenario. I can add that later today